osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -188,6 +200,7 @@ public override void OnResuming(IScreen last)
             base.OnResuming(last);
 
             (Background as BackgroundScreenDefault)?.Next();
+            DiscordRpc.updatePresence(Presence);
```

Comment:
This should not be here.

Reply:
Since it's repeated pretty much every time in `OnResuming()`, it should be moved to `OsuScreen.OnResuming()`.